rms_fe/src/views/ArticleDetailsPage/ArticleDetailsPage.tsx

```diff
@@ -35,7 +38,28 @@ const useExtractArticleFeatures = (id: string | number) => {
     })
 }
 
+const useSearchArticles = () => {
+    return useMutation<SearchResponse, Error, SearchBody>({
+        mutationFn: (searchBody: SearchBody) => {
+            return searchArticles(searchBody);
+        },
+        onSuccess: () => {
+            message.success("Recommendations generated successfully");
+        },
```

Comment:
imo the user can see that we have loaded stuff on the page, so this snackbar is unnecessary